manifests/v1beta1/installs/katib-external-db/katib-db-manager-deployment.yaml

```diff
@@ -8,8 +8,6 @@ spec:
       containers:
         - name: katib-db-manager
           env:
-            - name: DB_NAME
-              value: mysql
```

Comment:
Maybe we can introduce another env variable, like `DB_TYPE: mysql`, to define type of DB?